src/layout/MasterLayout.cpp

```diff
@@ -1331,10 +1331,13 @@ std::any CHyprMasterLayout::layoutMessage(SLayoutMessageHeader header, std::stri
         const auto PWINDOW = header.pWindow;
         const auto PNODE   = getNodeFromWindow(PWINDOW);
 
-        if (!PNODE)
+        if (!PWINDOW || !PNODE)
```

Comment:
!PWINDOW is redundant, nodes can't have a null window can they?

Reply:
right. changed.